Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-create.js

```diff
@@ -94,7 +130,7 @@ $(document).ready(function () {
 
             $('#remove-resource').removeAttr('disabled');
 
-            $('#resources .required-resource-field').each(function() {
+            $('#resources .required-resource-field').each(function () {
```

Comment:
Please be consistent with the type of quotes in JavaScript. Use only `'` or only `"`, but not both.